FreeRTOS/FreeRTOS-Kernel
# AutoRelease + Header Checker Upgrades

Description
-----------
 - decoupled kernel and core configs so they live in their respective repos. Need for next bullet
     - Check uses PRs ref, so users can update kernel config as needed
 - Add autorelease action

Test Steps
-----------
You can run this on your git forks for verification. See `release.py -h`

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS/pull/482


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
